.travis.yml

```diff
@@ -84,41 +101,43 @@ matrix:
         - clang-3.6
     env:
     - MATRIX_EVAL="CC=clang-3.6 && CXX=clang++-3.6"
-  - os: linux
-    addons:
-      apt:
-        update: true
-        sources:
-        - ubuntu-toolchain-r-test
-        - llvm-toolchain-precise-3.7
-        packages:
-        - *1
-        - clang-3.7
-    env:
-    - MATRIX_EVAL="CC=clang-3.7 && CXX=clang++-3.7"
-  - os: linux
-    addons:
-      apt:
-        update: true
-        sources:
-        - ubuntu-toolchain-r-test
-        - llvm-toolchain-precise-3.8
-        packages:
-        - *1
-        - clang-3.8
-    env:
-    - MATRIX_EVAL="CC=clang-3.8 && CXX=clang++-3.8"
-  - os: linux
-    addons:
-      apt:
-        update: true
-        sources:
-        - llvm-toolchain-trusty-4.0
-        packages:
-        - *1
-        - clang-4.0
-    env:
-    - MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
+#DISABLED to reduce the matrix a bit
```

Comment:
I'm assuming this is for your own testing. In practice, we can add an additional matrix entry (probably one will suffice) that will test `HUNTER_ENABLED=0`.